relevanceai/operations_new/ops.py

```diff
@@ -0,0 +1,152 @@
+from typing import Any, Dict, List, Optional
+from relevanceai.dataset.write import Write
+
+
+class Operations(Write):
+    def vectorize_text(
+        self,
+        fields: List[str],
+        models: Optional[List[Any]] = None,
+        filters: Optional[List[Dict[str, Any]]] = None,
+        chunksize: int = 100,
+    ):
+        """It takes a list of fields, a list of models, a list of filters, and a chunksize, and then it runs
+        the VectorizeOps function on the documents in the database
+
+        Parameters
+        ----------
+        fields : List[str]
+            List[str]
+        models : List[Any]
+            List[Any]
+        filters : List[Dict[str, Any]]
+            List[Dict[str, Any]]
+        chunksize : int, optional
+            int = 100,
+
+        Returns
+        -------
+            Nothing
+
+        """
+        from relevanceai.operations_new.vectorize.text.ops import VectorizeTextOps
+
+        models = ["all-MiniLM-L6-v2"]
```

Comment:
`if models is None` ? Also - please set default to be `sentence-transformers/all-mpnet-base-v2`